REQUIRE

```diff
@@ -1,2 +1,2 @@
 julia 0.4
-Compat 0.9.4
+Compat 0.9.5
```

Comment:
This is only needed for the test (for `String(take!(buf))`), but introducing a separate `test/REQUIRE` file just for this seemed a bit silly.